CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# dbt_reddit_ads v1.1.0
```

Comment:
Is there a reason this needs to be a breaking change? Given these updates I would assume this could be a patch update as v1.0.1?